mne/viz/backends/abstract_tool_bar.py

```diff
@@ -0,0 +1,38 @@
+"""Basic tool bar support."""
+
+# Authors: Guillaume Favelier <guillaume.favelier@gmail.com
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: Simplified BSD
+
+from abc import ABC, abstractmethod
+
+
+class _AbstractToolBar(ABC):
+    @abstractmethod
+    def _tool_bar_load_icons(self):
+        pass
```

Comment:
Can we try a variant of https://stackoverflow.com/questions/9202723/excluding-abstractproperties-from-coverage-reports#answer-41284681 ?